luigi/contrib/hadoop_jar.py

```diff
@@ -64,23 +64,40 @@ def __init__(self):
         pass
 
     def run_job(self, job):
+        ssh_config = job.ssh()
+        if ssh_config:
+            host = ssh_config.get("host", None)
+            key_file = ssh_config.get("key_file", None)
+            username = ssh_config.get("username", None)
```

Comment:
Sure, but `[]` would make the logic simpler right? Also you'll see form the stack trace exactly what attribute you're missing.